recipe/patch_yaml/python-eccodes.yaml

```diff
@@ -0,0 +1,7 @@
+if:
```

Comment:
Please add a comment explaining the patch right here in the yaml.

Reply:
You are fast :) Btw. I tried to run `python show_diff.py` as instructed in the template, but I get several error messages about invalid versions such as `4.10_1`, `1_0` that don't seem to have anything to do with my patch. Example traceback:
```
Traceback (most recent call last):
  File "/home/a002160/mambaforge/envs/conda-forge/lib/python3.11/concurrent/futures/process.py", line 256, in _process_worker
    r = call_item.fn(*call_item.args, **call_item.kwargs)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/a002160/src/git/conda-forge/conda-forge-repodata-patches-feedstock/recipe/./show_diff.py", line 96, in _process_subdir
    vals = do_subdir(subdir, raw_repodata_path, ref_repodata_path)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/a002160/src/git/conda-forge/conda-forge-repodata-patches-feedstock/recipe/./show_diff.py", line 75, in do_subdir
    new_index = _gen_new_index(raw_repodata, subdir)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/a002160/src/git/conda-forge/conda-forge-repodata-patches-feedstock/recipe/gen_patch_json.py", line 447, in _gen_new_index
    patch_yaml_edit_index(indexes[index_key], subdir)
  File "/home/a002160/src/git/conda-forge/conda-forge-repodata-patches-feedstock/recipe/patch_yaml_utils.py", line 467, in patch_yaml_edit_index
    raise e
  File "/home/a002160/src/git/conda-forge/conda-forge-repodata-patches-feedstock/recipe/patch_yaml_utils.py", line 442, in patch_yaml_edit_index
    if _test_patch_yaml(patch_yaml, record, subdir, fn):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/a002160/src/git/conda-forge/conda-forge-repodata-patches-feedstock/recipe/patch_yaml_utils.py", line 122, in _test_patch_yaml
    _keep = parse_version(record[k]) == parse_version(v)
            ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/a002160/mambaforge/envs/conda-forge/lib/python3.11/site-packages/packaging/version.py", line 52, in parse
    return Version(version)
           ^^^^^^^^^^^^^^^^
  File "/home/a002160/mambaforge/envs/conda-forge/lib/python3.11/site-packages/packaging/version.py", line 197, in __init__
    raise InvalidVersion(f"Invalid version: '{version}'")
packaging.version.InvalidVersion: Invalid version: '1_0'
"""

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/a002160/src/git/conda-forge/conda-forge-repodata-patches-feedstock/recipe/./show_diff.py", line 131, in <module>
    subdir, vals = fut.result()
                   ^^^^^^^^^^^^
  File "/home/a002160/mambaforge/envs/conda-forge/lib/python3.11/concurrent/futures/_base.py", line 456, in result
    return self.__get_result()
           ^^^^^^^^^^^^^^^^^^^
  File "/home/a002160/mambaforge/envs/conda-forge/lib/python3.11/concurrent/futures/_base.py", line 401, in __get_result
    raise self._exception
packaging.version.InvalidVersion: Invalid version: '1_0'
```

Reply:
you need to limit the patch to python-eccodes itself

Reply:
Ah ok. I thought that would happen via the filename. Got it now. Thanks!